frontend/lib/core/const/text_constants.dart

```diff
@@ -0,0 +1,169 @@
+// TODO: I copied this from tutorial, unsure if it's needed – remove if we prefer to use strings directly
```

Comment:
i think that better approach is to keep strings out of main code, so keeping it here is very good idea imo